### PR TITLE
Add support for trackerStats object, startDate, peersConnected

### PR DESCRIPTION
**Pull request by bilalghouri**
- Added trackerStats object which returns: host, leecherCount, seederCount, lastScrapeResult, lastAnnounceResult
"lastAnnounceResult" is an important feature that was missing. It returns the status of the tracker which the Tracker class does not return. It returns Success if the last announce attempt was successful or any error message returned by the tracker upon announce request. i.e Connection failed, Unregistered torrent, IP not registered with the tracker (in the case of private trackers) etc
More stats variables can be added to the return object easily upon request.
- Added startDate integer: unix timestamp of when the torrent was started
- Added peersConnected integer: number of peers connected

### CODE REVIEW

File: lib/Transmission/Model/Torrent.php [hunk 1]
```diff
@@ -46,6 +46,10 @@ class Torrent extends AbstractModel
     /**
      * @var integer
      */
+    protected $startDate;
```
Comment (kleiram) [inline lib/Transmission/Model/Torrent.php]: There should be an extra new-line here.

File: lib/Transmission/Model/trackerStats.php [hunk 2]
```diff
@@ -0,0 +1,127 @@
+<?php
+namespace Transmission\Model;
+
+/**
+ * @author Bilal Ghouri <bilalghouri@live.com>
+ */
+class trackerStats extends AbstractModel
```
Comment (kleiram) [inline lib/Transmission/Model/trackerStats.php]: The class name should be `CamelCase`.

**Comment by kleiram** [comment]
Hi there! Thanks for the PR! I added some comments to the code, if you could fix these issues that'd be great! Also, if you'd add tests that would be really good as well. Thanks again!

File: lib/Transmission/Model/trackerStats.php (hunk 2 above)
Comment (bilalghouri) [inline lib/Transmission/Model/trackerStats.php]: Done.

File: lib/Transmission/Model/Torrent.php (hunk 1 above)
Comment (bilalghouri) [inline lib/Transmission/Model/Torrent.php]: Done.

**Comment by bilalghouri** [comment]
Hello, if you could merge the pull request any time soon, that would be wonderful.

**Comment by kleiram** [comment]
Hi there, if you could some (rudimentary at least) tests, I'll merge them!

**Comment by bilalghouri** [comment]
Tests have been added. Please check. Thanks

**Comment by bilalghouri** [comment]
Hello, Sorry to be asking again, but can you please merge it so I can keep my build up to date on the servers?

**Comment by kleiram** [comment]
Merged now, the tests seemed to keep failing earlier.

**Pull request merged**